mifosng-android/src/main/java/com/mifos/mifosxdroid/login/LoginActivity.java

```diff
@@ -240,16 +264,7 @@ public void onLoginClick(Button button) {
         // Saving port
         PrefManager.setPort(et_port.getEditableText().toString());
         // Updating Services
-        if (!PrefManager.getInstanceDomain().equals(BaseUrl.API_ENDPOINT)) {
-            BaseApiManager.createService();
-        }
-
-        login(false);
-    }
-
-    private void login(boolean shouldByPassSSLSecurity) {
-        if (!validateUserInputs())
-            return;
+        BaseApiManager.createService();
```

Comment:
```if (!PrefManager.getInstanceDomain().equals(BaseUrl.API_ENDPOINT))``` this line telling us that user have changed the base URL or not, if not then we not need to do any task, do not need to write again base URL in Preference.